lib/testLauncher/SuitestLauncher.js

```diff
@@ -5,7 +5,7 @@ const {handleChildResult, handleLauncherError, followChildProcess} = require('..
 const envVars = require('../constants/enviroment');
 const suitest = require('../../index.js');
 const {validate, validators} = require('../validataion');
-const {snippets: log} = require('./launcherLogger');
+const {snippets: log, launcherLogger} = require('./launcherLogger');
 
 class SuitestLauncher {
```

Comment:
It seems that `runSutomatedSession` and `runInteractiveSession` have a lot in common. Should be refactored to avoid code duplication. If it's not too much work - do it in scope of this PR, because you added some more duplicated code here. Otherwise, please create a ticket and assign to me.